tests/framework/installer/ceph_manifests.go

```diff
@@ -1930,7 +1930,7 @@ spec:
     type: s3
     sslCertificateRef:
     port: ` + strconv.Itoa(port) + `
-    securePort:
+    securePort: 443
```

Comment:
I see now this is related to #4596. The securePort should still be optional. Could you also add a check if we're running on K8s 1.13 or lower before adding this port? Apparently the older k8s versions don't support `nullable`. The check would be something like this:
```
if !k8sh.VersionAtLeast("v1.14.0") {
  // add the port
}
```